package/MDAnalysis/lib/NeighborSearch.py

```diff
@@ -27,6 +27,51 @@
 
 from MDAnalysis.core.AtomGroup import AtomGroup
 
+class CoordinateNeighborSearch(object):
```

Comment:
This should not be here. Why are you bringing back code that was removed?

Are you coming from develop in the distant past? Better rebase against develop!